torch/_inductor/cudagraph_trees.py

```diff
@@ -937,8 +947,8 @@ def reconstruct_outputs(self):
         for i, (storage_info, metadata) in enumerate(
             zip(self.output_storage_alias, self.outputs_metadata)
         ):
-            if metadata is None:
-                outputs.append(None)
+            if not isinstance(metadata, dict):  # tensor metadata
```

Comment:
There are other places where the metadata is tested to be None, 

https://github.com/pytorch/pytorch/blob/9f3c06e3f9ae67c06134b963de86e1196b96c04b/torch/_inductor/cudagraph_trees.py#L986-L987